From 2a2c6bea4e79075ec4af35b9e92775ae988ec0ab Mon Sep 17 00:00:00 2001
From: Luciana Fujii Pontello <luciana@fujii.eti.br>
Date: Fri, 23 Dec 2011 02:43:52 -0200
Subject: [PATCH 2/2] configure.ac: Check for glib 2.30

gst-plugins-base requires glib >= 2.30 because of
g_cclosure_marshal_generic that was introduced in 2.30.
---
 configure.ac |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index 4c65068..f7cf5cf 100644
--- a/configure.ac
+++ b/configure.ac
@@ -260,7 +260,7 @@ AC_CHECK_FUNC(gethostbyname,,[AC_CHECK_LIB(nsl,gethostbyname)])
 dnl *** checks for dependency libraries ***
 
 dnl GLib is required
-AG_GST_GLIB_CHECK([2.22])
+AG_GST_GLIB_CHECK([2.30])
 
 ORC_CHECK([0.4.11])
 
-- 
1.7.5.4

